### PR TITLE
[node-manager][doc] Clarify master NodeGroup cri change

### PR DESCRIPTION
**Pull request by konstantin-axenov**
Signed-off-by: Konstantin Aksenov <konstantin.aksenov@flant.com>

## Description

Describe how to detect which node is updating right now and allow disruption for it.

## Why do we need it, and what problem does it solve?

There were several cases when users weren't able to update master without extra operations.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: chore
summary: Clarify master NodeGroup cri change.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/040-node-manager/docs/FAQ.md [hunk 1]
```diff
@@ -468,47 +470,19 @@ as described [here](#how-to-change-cri-for-nodegroup).
...
+   ```shell
+   kubectl get nodes -l node-role.kubernetes.io/control-plane="" -o json | jq '.items[] | select(.metadata.annotations."update.node.deckhouse.io/approved"=="") | .metadata.name' -r
+   ```
 
-     ```shell
-     for image in $(grep "image:" /etc/kubernetes/manifests/* | awk '{print $3}'); do
-       docker pull $image
-     done
-     ```
+1. Confirm the disruption of the master node that was discovered on previous step:
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ.md:479]:
```suggestion
1. Confirm the disruption of the master node that was discovered in the previous step:
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [2529](https://github.com/deckhouse/deckhouse/pull/2529) to the branch [release-1.36](https://github.com/deckhouse/deckhouse/tree/release-1.36) successful!

**Pull request merged**